examples/recipe-sourcing-contentful/.gitignore

```diff
@@ -0,0 +1,69 @@
+# Logs
```

Comment:
These hidden files probably shouldn't be included actually – there are gitignore and prettierrc files more broadly in the repo. It would also keep this example more focused on what's relevant to the recipe.

Reply:
Done in https://github.com/kevee/gatsby/commit/3fcf567d9566f63d2419875ef6bbe56f8a099f07